examples/quantization/brevitas/quantize_llm.py

```diff
@@ -95,40 +107,36 @@
     nsamples=128,
     seqlen=args.seqlen,
     split="validation",
-    device="cuda:0" if not args.cpu_offload else None,
+    device=args.device if not use_accelerate else None,
 )
 
-perplexity = compute_perplexity(
-    quantizer.model, validation_dataset, context_length=args.seqlen // 2, tokenizer=tokenizer
-)
+model = quantizer.model
 
+# Evaluation of the non-quantized model.
+if use_accelerate:
+    model = offload_model(model, qconfig.gpu_device_map, qconfig.cpu_device_map)
+perplexity = compute_perplexity(model, validation_dataset, context_length=args.seqlen // 2, tokenizer=tokenizer)
+if use_accelerate:
+    remove_hooks(model)
```

Comment:
I don't think this is needed, is it? `quantizer.quantize` should handle the hook removal / offloading, so that:

input is a model that uses accelerate -> output is a model that uses accelerate
input is NOT a model that uses accelerate -> output is NOT a model that uses accelerate